plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerColumnHandle.java

```diff
@@ -139,7 +139,12 @@ private static ValueSet stepValue(ColumnMetadata column)
         }
```

Comment:
Is there a use case for negative step ?

Reply:
Not really, dropped! I only did that because I was thinking "why not", but it doesn't make sense if all the other data is random. Even if there are multiple sequence columns in one table. I saved the commit in another branch just in case.